using-mobile-extensions/adobe-analytics/product-variable.md

```diff
@@ -12,14 +12,24 @@ To set the products variable, set a context data key to `&&products`, and set th
 
 #### Syntax <a id="syntax"></a>
 
-```text
+```java
 cdata.put("&&products", "Category;Product;Quantity;Price[,Category;Product;Quantity;Price]");
 ```
 
 #### Example <a id="example"></a>
 
-```text
-//create a context data dictionaryHashMap cdata = new HashMap<String, Object>();​// add products, a purchase id, a purchase context data key, and any other data you want to collect.// Note the special syntax for productscdata.put("&&products", ";Running Shoes;1;69.95,;Running Socks;10;29.99");cdata.put("myapp.purchase", "1");cdata.put("myapp.purchaseid", "1234567890");​// send the tracking call - use either a trackAction or TrackState call.// trackAction example:MobileCore.trackAction("purchase", cdata);// trackState example:MobileCore.trackState("Order Confirmation", cdata);
+```java
+//create a context data dictionary
+HashMap cdata = new HashMap<String, Object>();
+// add products, a purchase id, a purchase context data key, and any other data you want to collect.
+// Note the special syntax for products
+cdata.put("&&products", ";Running Shoes;1;69.95,;Running Socks;10;29.99");cdata.put("myapp.purchase", "1");
+cdata.put("myapp.purchaseid", "1234567890");
+// send the tracking call - use either a trackAction or TrackState call.
```

Comment:
Use same format for both the function names `use either a trackAction or trackState call.`